scout/config.py

```diff
@@ -4,7 +4,53 @@
 import sys
 from pathlib import Path
 from jsonschema import validate
-from scout.constants import FilePaths as fp
+
+
+class FilePaths:
+    # Package data:
+    _root_dir = Path(__file__).resolve().parents[0]  # package install dir
+    SUPPORTING_DATA = _root_dir / "supporting_data"
+    THERMAL_LOADS = SUPPORTING_DATA / "thermal_loads_data"
+    CONVERT_DATA = SUPPORTING_DATA / "convert_data"
+    STOCK_ENERGY = SUPPORTING_DATA / "stock_energy_tech_data"
+    TSV_DATA = SUPPORTING_DATA / "tsv_data"
+
+    # Non-package data:
+    _parent_dir = Path.cwd()  # parent dir of repo
+    ECM_DEF = _parent_dir / "ecm_definitions"
+    GENERATED = _parent_dir / "generated"
+    ECM_COMP = GENERATED / "ecm_competition_data"
+    EFF_FS_SPLIT = GENERATED / "eff_fs_splt_data"
+    INPUTS = _parent_dir / "inputs"
+    RESULTS = _parent_dir / "results"
+    PLOTS = RESULTS / "plots"
+    METADATA_PATH = INPUTS / "metadata.json"
+
+    # Store baseline FilePaths attributes
+    _base_paths = {
+        attr: value
+        for attr, value in locals().items()
+        if not callable(value) and not attr.startswith("__")
+    }
+
+    @classmethod
+    def set_paths(cls, paths_to_update: dict = None):
+        """Update class attributes to new values
+
+        Args:
+            paths_to_update (dict, optional): Update dictionary, where attributes are keys and
+                file paths are values. Defaults to None.
+        """
+
+        if paths_to_update:
+            for var, new_path in paths_to_update.items():
+                setattr(cls, var, new_path)
+
+    @classmethod
+    def reset_base_paths(cls):
+        """Reset filepath attributes to the baseline values defined at class definition"""
+        for attr, val in cls._base_paths.items():
+            setattr(cls, attr, val)
```

Comment:
Currently used in testing to reset file path variables after running `set_paths({"ecm_directory":"./custom_dir"`